src/data_collection.py

```diff
@@ -269,6 +269,98 @@ def print_individual_in_table(file_name):
                 dictionary[key]["FORMAT"],
             ]
         )
+
+    print(data_table)
+
+
+def get_testing_commit_info(json_file_name):
```

Comment:
I suggest merging this function with the calculate_individual_metrics function to avoid looping through the raw data twice. Additionally, this function has many aspects of that function that would cause some data to double or overwrite other data.